src/main/java/com/codeborne/selenide/CollectionCondition.java

```diff
@@ -1,16 +1,9 @@
 package com.codeborne.selenide;
 
-import com.codeborne.selenide.collections.ExactTexts;
-import com.codeborne.selenide.collections.ListSize;
-import com.codeborne.selenide.collections.SizeGreaterThan;
-import com.codeborne.selenide.collections.SizeGreaterThanOrEqual;
-import com.codeborne.selenide.collections.SizeLessThan;
-import com.codeborne.selenide.collections.SizeLessThanOrEqual;
-import com.codeborne.selenide.collections.SizeNotEqual;
-import com.codeborne.selenide.collections.Texts;
-import com.codeborne.selenide.collections.TextsInAnyOrder;
+import com.codeborne.selenide.collections.*;
```

Comment:
Why don't you run gradle `check` task locally?
This import changes spawn an error:
```
[ant:checkstyle] [ERROR] /home/travis/build/selenide/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:3: Using the '.*' form of import should be avoided - com.codeborne.selenide.collections.*. [AvoidStarImport]
```

Reply:
I've encountered with some checkstyle issue while running it locally. Now It's fixed. Thank you